Branch-SDK-TestBed/InitializationTestPlan.md

```diff
@@ -0,0 +1,92 @@
+# Test plan for different (re)initialization scenarios
+
+This is a test plan enlisting possible initialization scenarios. As CSMs
+find bugs arising from unforeseen initialization scenarios, this list should 
+be updated. Unless noted, initialization is expected to work in all scenarios.
+
+
+## Assumptions:
+* User followed [Android documentation](https://docs.branch.io/apps/android/)
+* Application can have multiple activities (all non-routing activities 
```

Comment:
A little confusing.  Maybe something like:
* Routed Activity:  An activity that can be deep linked to
* Non-Routed Activity:  An activity that may appear in the stack, but isn't one that can be directly linked to.

-- or something like that.  This isn't correct because elsewhere you refer to "RoutingActivity.onStart" and it isn't obvious what this means.   Splash for instance might not be deep-linkable, but it may very well want to call initSession.

Reply:
I think the terminology is getting convoluted. Taking your suggestion "RoutedActivity: An activity that can be deep linked to" is only true in Scenario 2 (at least according to my definition of "deeplinkable"). What I consider a d"eeplinkable Activity" is an Activity that is specified in the pending intent when creating a push notification + it calls `initSession`, in Scenario 1, only SplashActivity is deeplinkable and, in Scenario 2, both `SplashActivity` and `NextActivity` are deeplinkable.